README.md

```diff
@@ -31,11 +31,14 @@ npm install node-jq --save
 
 ## Usage
 
-Usually in your CLI with jq you run:
+#### jq example
```

Comment:
Yeah I'm not completely sure, I wanted to make the difference between `jq` and `node-jq` usage more obvious, that's why I added it.